server/container/util/src/test/java/org/apache/james/util/retry/naming/NamingExceptionRetryHandlerTest.java

```diff
@@ -31,24 +31,9 @@
 import org.junit.Before;
 import org.junit.Test;
 
-/**
```

Comment:
this commit 7ee73d3d99f4e45fe793a6b23b071ad45d41555b does more than just use @override, it should be separated